docs/versioned_docs/version-2.11.0/stateful-applications/minio.md

```diff
@@ -35,21 +35,21 @@ In this tutorial, Local PV volume will be provisioned on the node where the appl
 
 ### Install OpenEBS {#install-openebs}
 
-If OpenEBS is not installed in your K8s cluster, this can be done from [here](/docs/overview). If OpenEBS is already installed, go to the next step.
+If OpenEBS is not installed in your K8s cluster, this can be done from [here](/). If OpenEBS is already installed, go to the next step.
```

Comment:
Why link is not there ?

Reply:
Link is there. It is just `/` i.e., going to the immediate parent route. https://gallant-pasteur-5e438c.netlify.app/docs/

Reply:
There is no separate route for `overview`